Makefile

```diff
@@ -186,14 +186,18 @@ stylesheets: public/css/index.css
 
 .IGNORE: public/css/index.css
 public/css/index.css: $(STYLESHEETS)
-	lessc $< $@
+	@hash lessc > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		docker run --rm -v $(PWD):/app appleboy/node-less $< $@; \
+	else \
+		lessc $< $@; \
+	fi
```

Comment:
Modern frontend development includes a series of preprocessing to assets, like concatenating, automatically adding vendor prefixes, minifying css and js, precompiling gzip/brotli artifcats and more. The tools used for that all com from the Node.js ecosystem, so I see Node.js as a requirement for any serious web development.